app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Api::V1
+  class UsersController < ApiController
+
+    def current
+      render partial: 'user', locals: { user: current_user }
```

Comment:
@yunixon it's better to not make any render here or render `show` action (which we will have in future) and in `current.json.jbuilder` (or `show.json.jbuilder`) make `json.partial! 'user', user: current_user`